src/OpenTelemetry.Contrib.Instrumentation.Wcf/WcfInstrumentationOptions.cs

```diff
@@ -34,6 +35,18 @@ public class WcfInstrumentationOptions
             new BaggagePropagator(),
         });
 
+        /// <summary>
+        /// Gets or sets an action to enrich an Activity.
+        /// </summary>
+        /// <remarks>
+        /// <para><see cref="Activity"/>: the activity being enriched.</para>
+        /// <para>string: the name of the event. Will be one of the constants in <see cref="WcfEnrichEventNames"/>.
+        /// </para>
+        /// <para>object: the raw <see cref="Message"/> from which additional information can be extracted to enrich the activity.
+        /// </para>
+        /// </remarks>
+        public Action<Activity, string, object> Enrich { get; set; }
```

Comment:
it';d be good to add an entry in changelog.md as well for this new feature

Reply:
Done.